doc/examples/collidable/lib/main.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flame/game.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/components.dart';
+
+import 'package:flutter/material.dart' hide Image;
+
+import 'dart:ui';
+
+void main() async {
+  final game = MyGame();
```

Comment:
Shouldn't you be calling '  WidgetsFlutterBinding.ensureInitialized();' because MyGame has onLoad assets feature?

Reply:
This is just a draft PR, there is lots more coming. :)